src/components/Dropdown/SortDropdown/SortDropdown.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { ComponentProps, ComponentPropsWithRef } from 'react'
+import { type FC, type MouseEventHandler } from 'react'
+
+import { Button } from '../../Button'
+import { FormControl } from '../../FormControl'
+import { Cluster, Stack } from '../../Layout'
+import { Fieldset } from '../../NewFieldset'
+import { RadioButton } from '../../RadioButton'
+import { Section } from '../../SectioningContent'
+import { Select } from '../../Select'
+import { Dropdown } from '../Dropdown'
+import { DropdownCloser } from '../DropdownCloser'
+import { DropdownContent } from '../DropdownContent'
+import { DropdownScrollArea } from '../DropdownScrollArea'
+import { DropdownTrigger } from '../DropdownTrigger'
+
+import { useSortDropdown } from './useSortDropdown'
+
+import type { DecoratorsType } from '../../../types'
+
+type ArgsOnApply = {
+  field: string
+  order: 'asc' | 'desc'
+  newfields: ComponentProps<typeof Select>['options']
+}
+
+type Props = {
+  /** 並び替え項目 */
+  sortFields: ComponentProps<typeof Select>['options']
+  /** 並び順の初期値 */
+  defaultOrder: 'asc' | 'desc'
+  /** 適用時に発火するイベント */
+  onApply: (args: ArgsOnApply) => void
+  /** キャンセル時に発火するイベント */
+  onCancel?: MouseEventHandler<HTMLButtonElement>
+  decorators?: DecoratorsType<
+    | 'sortFieldLabel'
+    | 'sortOrderLabel'
+    | 'ascLabel'
+    | 'descLabel'
+    | 'applyButtonLabel'
+    | 'canselButtonLabel'
```

Comment:
`cancel`のtypoかもしれません！ページ内に複数あったのでgrepしていただくと良さそうです

```suggestion
    | 'cancelButtonLabel'
```